### ISSUE TITLE
Ability to run remote ssh scans with sudo privileges

### ISSUE THREAD

**Issue by mvazquezc** [OpenSCAP/openscap-daemon#84]
I've been testing and it works fine on newer versions of openscap (those which includes --sudo option).

Adding --sudo to the evaluation args works fine, but we need all the program logic to decide whether to add --sudo to the args or not. And probably we need to store a flag to tell daemon when to run scans with sudo privileges.

```python
ret.extend([config.oscap_ssh_path, '--sudo', host, str(port)])
```

Destination's machine mustn't require TTY to run sudo (scan fails if sudo requires a tty):

/etc/sudoers:
Defaults    requiretty -> #Defaults    requiretty
OR
Defaults:\<scanUser\>    !requiretty

It would be a "nice to have" feature whether allow users to run scans with sudo privileges or without them.

**Comment by mpreisler** [comment]
We could do it cleanly by adding `ssh+sudo://` to the target URI schemes.

**Comment by mvazquezc** [comment]
You're right. I'm going to give it a try tomorrow. I'll PR changes. Thank you!

**Comment by mpreisler** [comment]
Implemented in #85.
